BUILD.md

```diff
@@ -63,11 +63,21 @@ Distribution | Requires
 Alpine | cmake, g++, libexecinfo, make
 CentOS 8 | clang, cmake, diffutils, libatomic, libstdc++-static, make, zlib-devel
 Fedora | cmake, gcc-c++, libatomic, libstdc++-static, make
+Raspbian 64-bit | cmake, clang
 Rocky | clang, cmake, diffutils, libatomic, libstdc++-static, make, zlib-devel
 Ubuntu | cmake, g++, make
 
 Note that you only need to run `make libs` once the first time you build (or if the version of LLVM in the `lib/llvm/src` Git submodule changes).
 
+### 64-bit Raspbian
+
+```bash
+make libs
+make configure arch=armv8-a
+make build
```

Comment:
@kulibali same question as the other PR. does `arch` carry over from `configure` to subsequent operations? I assume no from looking at the Makefile.